time begin
time end
---
annotated_map_msgs/TaggedPolygonalMap map